calcom/cal.com
# feat: troubleshooter with weekly view (V2)

## What does this PR do?
Introduces a new troubleshooter than is easier to understand and user than the previous ones

Fixes: https://github.com/calcom/cal.com/pull/12280

## How to test?
- Create a schedule and assign it to an event type
- Add a date override (unavailable all day one)
- Add a calendar 
- Load troubleshooter 
- Ensure data is correct 



Screenshot:
(Excuse the translation not loading)
![CleanShot 2023-11-09 at 10 14 52@2x](https://github.com/calcom/cal.com/assets/55134778/8d5bc5c4-1c2c-46a1-a3d6-92c70a0b2c0a)
